pkg/apis/v1alpha1/register.go

```diff
@@ -136,7 +133,6 @@ func init() {
 		LabelInstanceAcceleratorName,
 		LabelInstanceAcceleratorManufacturer,
 		LabelInstanceAcceleratorCount,
-		LabelInstanceAcceleratorMemory,
```

Comment:
Just to confirm, is this being removed in favor of LabelInstanceGPUMemory?

Reply:
GPU memory was always being used. The accelerator info that is returned by the EC2 api does not have any information on memory

Reply:
See an example response for an inferentia instance

```console
"InferenceAcceleratorInfo": {
                "Accelerators": [
                    {
                        "Count": 1,
                        "Name": "Inferentia",
                        "Manufacturer": "AWS"
                    }
                ]
            },
```